kolibri/plugins/coach/assets/src/views/common/HeaderTabs/index.vue

```diff
@@ -1,9 +1,10 @@
 <template>
 
   <div
+    v-show="enablePrint || $mediaType !== 'print'"
```

Comment:
why are both conditions necessary?

Reply:
There is styling to show the title of the active tab in print media, which is required for certain reports, but not most.